src/childinfo.c

```diff
@@ -31,9 +31,8 @@
 #include <unistd.h>
 
 typedef struct {
-    int process_type;           /* AOF or RDB child? */
-    int on_exit;                /* COW size of active or exited child */
-    size_t cow_size;            /* Copy on write size. */
+    int information_type;            /* Type of information */
+    size_t info;               /* Information about the process. */
```

Comment:
As per the comment below, it seems like a better format is to make this into a union for the different types of information we can send. Then we can send a single packet with all the information.